app/models/message.rb

```diff
@@ -0,0 +1,16 @@
+class Message < ApplicationRecord
+  belongs_to :sender, :foreign_key => :sender_id, class_name: 'User'
+  belongs_to :recipient, :foreign_key => :recipient_id, class_name: 'User'
```

Comment:
In these two lines, you can delete the `:foreign_key` definitions. The other two pieces that you have already point to the correct information.

For more, you can read through this resource: https://www.theodinproject.com/courses/ruby-on-rails/lessons/active-record-associations